zeppelin-zengine/src/main/java/org/apache/zeppelin/util/Util.java

```diff
@@ -29,6 +30,9 @@
   private static final String PROJECT_PROPERTIES_VERSION_KEY = "version";
   private static final String GIT_PROPERTIES_COMMIT_ID_KEY = "git.commit.id.abbrev";
   private static final String GIT_PROPERTIES_COMMIT_TS_KEY = "git.commit.time";
+  private static final String ZEPPELIN_NOTE_FILE_EXTENSION = "zpln";
+  private static final int MAX_ZEPPELIN_NOTE_FILE_LENGTH = 127;
```

Comment:
after short review of max filename lengths on various filesystems 255 seemed to be correct number of chars. However some mobile platforms (e.g. android) have problems with having more than 127 chars, so i thought it would be safer. any other opinion on this point is welcome